docs/getting-started.md

```diff
@@ -90,39 +90,46 @@ Explore our CLI to find more queries you can run.
 
 ### Submitting a transaction
 
+An account has an address which is determined by the key pair
 
-1. Generate an account
 
-    You need an account to author transactions.
-    An account is a randomly generated key pair identified by a name of your choice.
-    To generate an account, run:
+1. Generate a key pair
+
+    You need a key pair to author transactions.
```

Comment:
We didn't get rid of the `account` notion completely, it's still used below. I propose to keep it properly introduced.
```suggestion
1. Generate an account and its key pair

    You need an account to author transactions.
    An account doesn't need to be explicitly created, as soon as you create a key pair,
    you can start using a new account controlled with it.
```

Reply:
I think this is properly addressed in this running discussion: https://github.com/radicle-dev/registry.radicle.xyz/pull/37#discussion_r419305153